brython/async/ajax/simple.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html >
```

Comment:
This did not work for me in Firefox on Manjaro - not sure why. I'll try it on another machine

Reply:
Good catch! It was a bug. The link to the CDN for brython.min.js was incorrect, the 9 and 0 were flipped, 3.0.9 instead of 3.9.0. I introduced that when migrating from 3.8.9 to 3.9.0 prior to the second round of TR. Brython is now 3.9.1, but I 
want to keep the links pointing to 3.9.0 as I don't want to introduce unexpected errors now the TR is complete.